grails-app/controllers/org/pih/warehouse/reporting/ReportController.groovy

```diff
@@ -303,14 +303,6 @@ class ReportController {
         render(template: "dataTableDialog", model: [url: url])
     }
 
-    def showShippingReport(ChecklistReportCommand command) {
-        command.rootCategory = productService.getRootCategory()
-        if (!command?.hasErrors()) {
-            reportService.generateShippingReport(command)
```

Comment:
Is this still used? Are these other reports that use this service method still used or perhaps we could also remove them?

Reply:
@alannadolny now i see the description :see_no_evil: if it is is used then lets leave it as is